src/Proto.Mailbox/Mailbox.cs

```diff
@@ -187,10 +192,20 @@ public void Start()
             }
             catch (Exception e)
             {
-                await _invoker.EscalateFailureAsync(e, msg);
+                HandleProcessMessageException(e, msg);
             }
             return true;
         }
+        
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Does it require the JIT to do it as a hotspot?  might not happen for various reasons.

Would you rather the logic just be copy pasted into the method then?